x-pack/plugins/secops/public/components/timeline/body/helpers.test.ts

```diff
@@ -189,17 +189,7 @@ describe('helpers', () => {
     });
 
     test('it returns true for when notes is non-empty', () => {
```

Comment:
nit, and this change was not part of your PR, but `for when` should just be `when`